lessons/Early-Math/distributive-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+= The Distributive Property
+
+@description{Students develop an understanding of the Distributive Property by studying equivalent Circles of Evaluation. They then apply their understanding of the Distributive Property in a variety of contexts.}
+
+@lesson-prereqs{identity-property}
+
+@keywords{Circle of Evaluation, }
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Recognize how distribution results in equivalent expressions.
+* Apply the Distributive Property to expressions with numbers and variables.
+* Use distribution to efficiently complete mental computations.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Distributive Property and Circles of Evaluation.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* A common approach to teaching the Distributive Property is by using an area model. This approach works well for students who are comfortable visualizing and computing area - but may leave other students confused.
+* We introduce the Distributive Property by inviting students to look at patterns in _structure_, and then later apply what they know using the area model.
+
+|===
+
+== The Distributive Property @duration{30 minutes}
+
+=== Overview
+
+Students develop an understanding of the Distributive Property by looking at equivalent Circles.
+
+=== Launch
+
+Elise and Dani are discussing the two Circles, below.
+
+[.embedded, cols="^.^3,^.^3", grid="none", stripes="none" frame="none"]
+|===
+| @show{(coe '(* 9 (+ 40 6)))} | @show{(coe '(+ (* 9 40) (* 9 6)))}
+|===
+
```

Comment:
I wonder if this launch would be better served with a notice and wonder than a debate? I usually use debates between two characters as something for students to respond to in writing on assessments and homework assignments rather than as openers.